CHANGELOG.md

```diff
@@ -4,10 +4,18 @@
 
 ### Features
 
-- **auth**: Fix cancelling hostedUI returning a generic error (#982)
 - Support lazy load and pagination for API (#1009)
 - Separate DataStore List logic out to list provider (#1000)
 
+### ⚠ BREAKING CHANGES
+
+- **auth**: Fix cancelling hostedUI returning a generic error (#982). User cancelled error returned from `Amplify.Auth.signInWithWebUI` is changed from `SFAuthenticationError.canceledLogin` to `AuthError.service(_, _, AWSCognitoAuthError.userCancelled)`.
```

Comment:
We flagged this as a bug fix, not a breaking change, which is why we didn't do a major version bump. We should note that it changes the behavior though.

```suggestion
### Behavior Change

- **Auth**: Fix cancelling hostedUI returning a generic error (#982).

    When a user cancels the sign in flow from `Amplify.Auth.signInWithWebUI`, the callback will now deliver an error of type `AuthError.service`, with an underlying error of `AWSCognitoAuthError.userCancelled`. Previously, the error was a `SFAuthenticationError.canceledLogin`. The `AuthError.service` error type will be returned regardless of whether the underlying HostedUI operation was performed via an SFAuthenticationSession or an ASWebAuthenticationSession.
    
    You can detect the user cancellation case by `switch`ing on the error, as in:
    
    <add error switch example in a ` ` `swift backtick>  
```